pkg/codegen/go/gen.go

```diff
@@ -4512,15 +4514,33 @@ func packageName(pkg *schema.Package) string {
 	return goPackage(root)
 }
 
-func GeneratePackage(tool string, pkg *schema.Package) (map[string][]byte, error) {
+func GeneratePackage(tool string,
+	pkg *schema.Package,
+	localDependencies map[string]string,
+) (map[string][]byte, error) {
 	if err := pkg.ImportLanguages(map[string]schema.Language{"go": Importer}); err != nil {
 		return nil, err
 	}
 
 	var goPkgInfo GoPackageInfo
 	if goInfo, ok := pkg.Language["go"].(GoPackageInfo); ok {
 		goPkgInfo = goInfo
+	} else {
+		goPkgInfo = GoPackageInfo{}
```

Comment:
This else isn't needed. goPkgInfo is already zero initialised.

Reply:
Nice catch 🙇‍♂️